tests/BasicTest.php

```diff
@@ -36,7 +36,7 @@ public function testSimple1()
     {
         $this->c->rule('name', ['required', ['lengthMin', 3]]);
 
-        $err = $this->m->unload()->set('name', 'a')->validate();
+        $err = $this->m->createEntity()->set('name', 'a')->validate();
```

Comment:
private function `createEntityForValidation` would be nice

Reply:
didn't see this, i don't get your point of making that private function, in theory, is needed only for tests

Reply:
not sure what I meant here :), fix conflict and we can merge this PR for 3.0